docs/_includes/head.html

```diff
@@ -15,4 +15,5 @@
 	<link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" />
 	<link rel="stylesheet" href="css/styles.css">
 	<link rel="stylesheet" href="css/docs.css">
+	<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/prismjs@1.17.1/themes/prism.min.css">
```

Comment:
I think cloudflare is pretty popular, haven't heard of jsdelivr before: https://cdnjs.com/libraries/prism. Maybe use it from there?